lib/models/entity.py

```diff
@@ -14,6 +18,61 @@ def __init__(self, name: str, description: str, damage: int):
         super().__init__(name, description)
 
 
+class Room(Entity):
+
+    class Exit(Entity):
+        def __init__(self, name, description, destination, exit_type: ExitType):
+            self.destination = destination
+            self.type = exit_type
+            super().__init__(name, description)
+
+
+    class DescriptionItem():
+        '''Similar to Entity, but distinct in that you'll never be able to pick up
+        a description item. It exists just for flavour text. Every noun in a room's
+        description should have a corresponding description item.
+        '''
+        def __init__(self, name: str, aliases: List[str], description: str):
+            self.name = name
+            self.aliases = aliases
+            self.description = description
+
+
+    def __init__(self, 
+                 name: str, 
+                 description: str,
+                 description_items: List[DescriptionItem] = None,
+                 exits: List[Exit] = None,
+                 light_level: LightLevel = LightLevel.BRIGHT,
```

Comment:
Did you forget to add a file where these are defined?

Reply:
`lib.constants`